cfme/intelligence/reports/widgets/chart_widgets.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+"""Page model for Cloud Intel / Reports / Dashboard Widgets / Charts"""
+from widgetastic_manageiq import Calendar
+from widgetastic_patternfly import BootstrapSelect
+from utils.appliance.implementations.ui import navigator
+from . import (Widget, DashboardWidgetFormCommon, NewDashboardWidget, EditDashboardWidget,
+    NewDashboardWidgetView, EditDashboardWidgetView)
+
+
+class ChartWidgetFormCommon(DashboardWidgetFormCommon):
+    # Chart Report
+    filter = BootstrapSelect("repfilter_typ")
+    # Timer
+    run = BootstrapSelect("timer_typ")
+    every = BootstrapSelect("timer_hours")
+    time_zone = BootstrapSelect("time_zone")
+    starting_date = Calendar("miq_date_1")
+    starting_hour = BootstrapSelect("start_hour")
+    starting_minute = BootstrapSelect("start_min")
+
+
+class NewChartWidgetView(NewDashboardWidgetView, ChartWidgetFormCommon):
+    pass
+
+
+class EditChartWidgetView(EditDashboardWidgetView, ChartWidgetFormCommon):
+    pass
+
+
+class ChartWidget(Widget):
```

Comment:
Can you cahnge the Widget class name to ManageIQWidget or something? This confuses with WT widget :)